src/hacbs/components/Environment/__tests__/EnvironmentCard.spec.tsx

```diff
@@ -10,6 +10,10 @@ jest.mock('@openshift/dynamic-plugin-sdk', () => ({
   useFeatureFlag: jest.fn(),
 }));
 
+jest.mock('@openshift/dynamic-plugin-sdk-utils', () => ({
+  commonFetch: jest.fn(),
```

Comment:
I'm not sure why this test suddenly started failing with my PR here. The SDK update should not impact `commonFetch` - it is still exposed through lib-utils: https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/lib-utils/src/index.ts#L15

Was this broken in main?

Reply:
No, it wasn't breaking in main.